.github/workflows/test.yml

```diff
@@ -62,13 +60,13 @@ jobs:
           export OMP_NUM_THREADS=2
           export OPENBLAS_NUM_THREADS=2
           export MPI_DISABLE=1
-          python3 setup.py test
+          python3 -m unittest discover
 
       - name: Run MPI Tests
         run: |
           export OMP_NUM_THREADS=1
           export OPENBLAS_NUM_THREADS=1
-          mpirun -np 2 python3 setup.py test
+          python3 -m unittest discover
```

Comment:
Can we restore the `mpirun` here?  Even 2 processes is enough to catch some problems.